src/main/resources/bin/validate-refs.bat

```diff
@@ -51,7 +51,7 @@ set LIB_DIR=%PARENT_DIR%\lib
 if exist "%LIB_DIR%\validate-@project.version@.jar" (
 set VALIDATE_JAR=%LIB_DIR%\validate-@project.version@.jar
 ) else (
-echo Cannot find VTool jar file (validate-@project.version@.jar) in %LIB_DIR%
+echo "Cannot find VTool jar file (validate-@project.version@.jar) in %LIB_DIR%""

```

Comment:
Is the double `""` at the end of this line correct?

(And is the whole file using DOS line endings? I see a `^M` here, but just here, but that may be a GitHub display artifact.)

Reply:
@nutjob4life fixed. not sure about the carriage return? I don't feel that I am ever successful in seeing these.

Reply:
@jordanpadams you and me both 😬